doc/Rules.md

```diff
@@ -1571,6 +1571,8 @@ Tags: code, language
 
 Aliases: fenced-code-language
 
+Parameters: allow_list (array of string; default `null`)
```

Comment:
What about calling it "valid_languages"? "Allow" doesn't seem as appropriate here because this rule is requiring one. "Languages" instead of "list" makes it clearer what is expected.